ui/Tabs.tsx

```diff
@@ -0,0 +1,58 @@
+import { Children, ReactNode } from "react";
+import { twMerge } from "tailwind-merge";
+import { Tab } from "@headlessui/react";
+
+interface TabsProps {
+  children: ReactNode;
+  [key: string]: any;
+}
+
+export const TabGroup = ({ children, ...props }: TabsProps) => {
```

Comment:
I think `TabsProps` can be changed to `React.HTMLAttributes<HTMLDivElement>`

```suggestion
export const TabGroup = ({ children, ...props }: React.HTMLAttributes<HTMLDivElement>) => {
```

Reply:
this one has to be the TabsProps, as it gets type error with React.HTMLAttributes

<img width="836" alt="image" src="https://github.com/SwaprHQ/swapr-ui/assets/5664434/aa9e57f4-dbb8-4ee1-9e25-eb34006b0d8d">

Reply:
React.PropsWithChildren<typeof Tab.Group> Seems to work well here

Reply:
I was wrong, this didn't help because onChange has different types.

```suggestion
export const TabGroup = ({ children, ...props }: React.PropsWithChildren<TabProps<React.ElementType>>) => {
```

This works, where we export `TabProps`